pkg/controllers/apps/description/description.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"fmt"
 	"reflect"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
seems using `runtime.Object` ("k8s.io/apimachinery/pkg/runtime") is fine.

Reply:
Yes, i have modified. And removed the dependency of the import of controller-runtime.


`UpdateFunc: func(old, cur interface{}) {}` compare the spec info of resources on child cluster, if the spec content changed the related `Descrption `wil be enqueued. 

I tested with namespace + deployment, and it worked fine.

But maybe not well when resource has multi-controller,  Managedfield may help to filter the unnecessary enqueue event, which need more discussion.